docs/DeveloperGuide.adoc

```diff
@@ -10,7 +10,9 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level4/tree/master
+
+:repoURL: https://github.com/CS2103JAN2018-W14-B1/main/tree/master
```

Comment:
it would be nice if you listed out your user profile clearly and explicitly here.